### DISCUSSION TITLE
npm ERR! 404 Not Found - GET https://registry.npmjs.org/hardat - Not found

### DISCUSSION THREAD

**Discussion by fgprayogo** [NomicFoundation/hardhat#2450] [Troubleshooting] [answered]
What happen with this error?

<img width="543" alt="Screen Shot 2022-03-03 at 22 19 13" src="https://user-images.githubusercontent.com/39540958/156595070-077fadec-374b-4c95-bfd9-93f8c06dc23c.png">

EDIT : silly mistake it should hardhat not hardhat

**Comment by fvictorio** [comment, marked as answer]
There is a typo in your command: you typed `hardat` instead of `hardhat`

(I know you figured it out, just responding here to be able to mark this as answered).
